tools/cpp/windows_cc_configure.bzl

```diff
@@ -227,9 +228,38 @@ def find_msvc_tool(repository_ctx, vc_path, tool):
     tool_path = vc_path + "\\bin\\amd64\\" + tool
 
   if not repository_ctx.path(tool_path).exists:
-    auto_configure_fail(tool_path + " not found, please check your VC++ installation.")
+    return None
+
   return tool_path
 
+def _check_vc_installation(repository_ctx, vc_path):
+  """Check if the VC++ installation is valid."""
+  if vc_path == "visual-studio-not-found":
```

Comment:
It's easy to mistype this string and then this check would always fail. Could you change the code so `vc_path` is None when VC is not found?